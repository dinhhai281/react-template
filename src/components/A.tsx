import React from 'react';

export const A = () => <div>Hello</div>;
