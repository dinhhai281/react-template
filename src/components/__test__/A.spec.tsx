import { A } from '@components/A';
import { render } from '@testing-library/react';
import React from 'react';

describe('A Component', () => {
  test('should match snapshot', () => {
    const { container } = render(<A />);
    expect(container).toMatchSnapshot();
  });
});
