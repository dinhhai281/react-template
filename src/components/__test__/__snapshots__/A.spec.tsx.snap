// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A Component should match snapshot 1`] = `
<div>
  <div>
    Hello
  </div>
</div>
`;
